deps/tools/common.mk

```diff
@@ -27,7 +27,11 @@ ifneq ($(VERBOSE), 0)
 CMAKE_COMMON += -DCMAKE_VERBOSE_MAKEFILE=ON
 endif
 # The call to which here is to work around https://cmake.org/Bug/view.php?id=14366
+ifeq ($(BUILD_OS),WINNT)
+CMAKE_COMMON += -DCMAKE_C_COMPILER="$(CC_BASE)"
```

Comment:
According to the listed cmake bug it was fixed in 3.0 so that should be perfectly fine.